backend/src/main/java/io/papermc/hangar/db/dao/internal/table/versions/downloads/ProjectVersionDownloadsDAO.java

```diff
@@ -68,8 +68,8 @@ public interface ProjectVersionDownloadsDAO {
         WHERE pvpd.version_id = (SELECT id
```

Comment:
this whole inner query is not needed, right? it was only used to get the id, but we now have the id as param